<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Money Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="centralWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>861</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="transactsTab">
     <attribute name="title">
      <string>Übersicht</string>
     </attribute>
     <widget class="QDateEdit" name="startDateEdit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>110</width>
        <height>24</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2021</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="date">
       <date>
        <year>2021</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
     <widget class="QPushButton" name="allTransactsButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Alle Transaktionen</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="endDateEdit">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>110</width>
        <height>24</height>
       </rect>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="date">
       <date>
        <year>2021</year>
        <month>2</month>
        <day>1</day>
       </date>
      </property>
     </widget>
     <widget class="QTableView" name="transactsTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>831</width>
        <height>451</height>
       </rect>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLineEdit" name="filterLineEdit1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="filterLineEdit2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="filterLineEdit3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="filterLineEdit4">
      <property name="geometry">
       <rect>
        <x>331</x>
        <y>90</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="filterLineEdit5">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>90</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="filterLineEdit6">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>90</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="filterLineEdit7">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>90</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="filterLineEdit8">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>90</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="importTab">
     <attribute name="title">
      <string>Importieren</string>
     </attribute>
     <widget class="QPushButton" name="activateImportButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Import aktivieren</string>
      </property>
     </widget>
     <widget class="QLabel" name="importPathLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>301</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ordner über den neue Dateien importiert werden:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="browseFolderEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>341</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="browseFolderButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>113</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Durchsuchen</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>oder</string>
      </property>
     </widget>
     <widget class="QLabel" name="importPathLabel_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Neue Datei einzeln importieren:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseFileButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>170</y>
        <width>113</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Durchsuchen</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="browseFileEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>341</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>/Users/lorisschmit1/Downloads/Export_Mouvements_Current acc. Green Code 18-30 Study.csv</string>
      </property>
     </widget>
     <widget class="QPushButton" name="importButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Importieren</string>
      </property>
     </widget>
     <widget class="QPushButton" name="taggingEnterButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>440</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Bestätigen</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="taggingTableWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>401</width>
        <height>51</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>449</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Kategorie der Transaktion:</string>
      </property>
     </widget>
     <widget class="QLabel" name="importProgressLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>449</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="taggingLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="tagReferenceEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="saveTagButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>480</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Speichern</string>
      </property>
     </widget>
     <widget class="QPushButton" name="notSaveTagButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>480</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Nein</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="monthTab">
     <attribute name="title">
      <string>Monat</string>
     </attribute>
     <widget class="QLineEdit" name="monthEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Monat</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="yearEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Jahr</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CreateBalanceButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Monatliche Bilanz erstellen</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AssignPaybackButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Rückzahlungen zuweisen</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showBalanceSheetButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Monatliche Bilanz zeigen</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="yearTab">
     <attribute name="title">
      <string>Jahr</string>
     </attribute>
     <widget class="QLineEdit" name="yearlySheetEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Jahr</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showYearlyBalanceSheetButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Jährliche Bilanz zeigen</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createYearlyBalanceButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Jährliche Bilanz erstellen</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="yearlySheetCreationProgressBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>291</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="progressBarLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="accountsTab">
     <attribute name="title">
      <string>Konten</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Konten:</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="createAccountButton">
         <property name="text">
          <string>Konto hinzufügen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteAccountButton">
         <property name="text">
          <string>Konto löschen</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QListWidget" name="accountsListWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>371</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>885</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
